android-lint.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<lint>
+</lint>
```

Comment:
Why add it when it's empty?

Reply:
If we do configure issues in the future the file is already in the correct place